.github/workflows/unit-tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Run unit tests
         run: |
-          uv run -p 3.10.16 --with . --with ".[dev]" --with ".[test]" pytest -s -v tests/unit/ --junitxml=pytest-report.xml
+          ./scripts/unit-tests.sh
```

Comment:
nit: better to use `sh ./scripts/unit-tests.sh`

Reply:
Why? (It's already in shebang line, and the script has +x permission; what am I missing?)